functions.php

```diff
@@ -1,17 +1,23 @@
 <?php
+
 /**
- * Timber starter-theme
- * https://github.com/timber/starter-theme
+ * Functions and definitions
+ *
+ * @link https://developer.wordpress.org/themes/basics/theme-functions/
+ * @link https://github.com/timber/starter-theme
  */
 
+namespace Timber\StarterTheme;
+
+use Timber\Timber;
+
 // Load Composer dependencies.
 require_once __DIR__ . '/vendor/autoload.php';
-
 require_once __DIR__ . '/src/StarterSite.php';
 
-Timber\Timber::init();
+Timber::init();
 
 // Sets the directories (inside your theme) to find .twig files.
-Timber::$dirname = [ 'templates', 'views' ];
+Timber::$dirname = ['templates', 'views'];
```

Comment:
Why not remove that and just stick to **views**?

Reply:
Agreed, I will remove this!